crates/holochain/src/sweettest/sweet_conductor.rs

```diff
@@ -124,13 +122,10 @@ impl SweetConductor {
         rendezvous: Option<DynSweetRendezvous>,
     ) -> SweetConductor {
         // Automatically add a test app interface
-        handle
```

Comment:
Commented this out while I was coding because this function call had to change. But then nothing is broken... so I guess it wasn't being used. May as well go, saves allocating a whole lot of ports on CI.

Reply:
=D